content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -774,9 +785,13 @@ node/mynode cordoned
 node/mynode drained
 ```
 
+{{< note >}}
+`kubectl` cannot set extra fields or impersonate UIDs.
+{{< /note >}}
+
 To impersonate a user, group, or set extra fields, the impersonating user must
```

Comment:
```suggestion
To impersonate a user, group, uid, or extra fields, the impersonating user must
```

Reply:
```suggestion
To impersonate a user, group, user UUID, or extra fields, the impersonating user must
```
or
```suggestion
To impersonate a user, group, user identifier (UID), or extra fields, the impersonating user must
```
or
```suggestion
To make any request involving impersonation, the user attempting impersonation must
```
(I'm not sure which is more appropriate)

Reply:
> To impersonate a user, group, user identifier (UID), or extra fields, the impersonating user must

That sounds the best to me.